### PR TITLE
updating syntax highlighting 

### PR DESCRIPTION
**Pull request by elliottburris**

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
```
Comment (rothfels) [inline release.sh:1]: I don't see this file being referenced anywhere, are you sure you're using it?

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "packages/*"
   ],
   "scripts": {
+    "generate": "npm --workspace vscode-mdx run generate",
```
Comment (rothfels) [inline package.json:9]: this needs to say `vscode-glass`

**Review by rothfels — APPROVED** [review]
small changes requested

**Review by rothfels — APPROVED** [review]
small changes requested

File: release.sh (hunk 1 above)
Comment (elliottburris) [reply release.sh:1]: removed

File: package.json (hunk 2 above)
Comment (elliottburris) [reply package.json:9]: fixed, will push

**Pull request merged**